pytorch_lightning/trainer/deprecated_api.py

```diff
@@ -6,138 +6,190 @@
+        rank_zero_warn(
+            "Attribute `num_gpu_nodes` has renamed to `num_nodes` since v0.5.0"
+            " and this method will be removed in v0.8.0",
+            DeprecationWarning,
+        )
         return self.num_nodes
 
     @num_gpu_nodes.setter
     def num_gpu_nodes(self, num_nodes):
         """Back compatibility, will be removed in v0.8.0"""
-        rank_zero_warn("Attribute `num_gpu_nodes` has renamed to `num_nodes` since v0.5.0"
-                       " and this method will be removed in v0.8.0", DeprecationWarning)
+        rank_zero_warn(
+            "Attribute `num_gpu_nodes` has renamed to `num_nodes` since v0.5.0"
+            " and this method will be removed in v0.8.0",
+            DeprecationWarning,
+        )
         self.num_nodes = num_nodes
 
     @property
     def gradient_clip(self):
         """Back compatibility, will be removed in v0.8.0"""
-        rank_zero_warn("Attribute `gradient_clip` has renamed to `gradient_clip_val` since v0.5.0"
-                       " and this method will be removed in v0.8.0", DeprecationWarning)
+        rank_zero_warn(
+            "Attribute `gradient_clip` has renamed to `gradient_clip_val` since v0.5.0"
+            " and this method will be removed in v0.8.0",
+            DeprecationWarning,
+        )
         return self.gradient_clip_val
 
     @gradient_clip.setter
     def gradient_clip(self, gradient_clip):
         """Back compatibility, will be removed in v0.8.0"""
-        rank_zero_warn("Attribute `gradient_clip` has renamed to `gradient_clip_val` since v0.5.0"
-                       " and this method will be removed in v0.8.0", DeprecationWarning)
+        rank_zero_warn(
+            "Attribute `gradient_clip` has renamed to `gradient_clip_val` since v0.5.0"
+            " and this method will be removed in v0.8.0",
+            DeprecationWarning,
+        )
         self.gradient_clip_val = gradient_clip
 
     @property
     def max_nb_epochs(self):
         """Back compatibility, will be removed in v0.8.0"""
-        rank_zero_warn("Attribute `max_nb_epochs` has renamed to `max_epochs` since v0.5.0"
-                       " and this method will be removed in v0.8.0", DeprecationWarning)
+        rank_zero_warn(
+            "Attribute `max_nb_epochs` has renamed to `max_epochs` since v0.5.0"
+            " and this method will be removed in v0.8.0",
+            DeprecationWarning,
+        )
         return self.max_epochs
 
     @max_nb_epochs.setter
     def max_nb_epochs(self, max_epochs):
         """Back compatibility, will be removed in v0.8.0"""
-        rank_zero_warn("Attribute `max_nb_epochs` has renamed to `max_epochs` since v0.5.0"
-                       " and this method will be removed in v0.8.0", DeprecationWarning)
+        rank_zero_warn(
+            "Attribute `max_nb_epochs` has renamed to `max_epochs` since v0.5.0"
+            " and this method will be removed in v0.8.0",
```

Comment:
```suggestion
            "Attribute `max_nb_epochs` has renamed to `max_epochs` since v0.5.0 and this method will be removed in v0.8.0",
```

Reply:
Can't change that - it is black formatter

Reply:
And also for next comments also. It is just master branch with black applied. I guess we already specified most of the options